10.9-libcxx/stable/main/finkinfo/devel/meson-py.info

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; tab-width: 4 -*-
+Info4: <<
+Package: meson-py%type_pkg[python]
+Version: 0.53.2
+Revision: 1
+Maintainer: Leigh Smith <leigh@leighsmith.com>
+Type: python (3.6 3.7 3.8)
+# setuptools needed because meson uses pkg_resources package.
+Depends: <<
+	python%type_pkg[python],
+	setuptools-tng-py%type_pkg[python]
+<<
+Source: https://files.pythonhosted.org/packages/49/d7/38a71de6adaa7a25a72587e69429ce78e137f4d545c7516341aa081f43e7/meson-%v.tar.gz
+Source-Checksum: SHA256(3e8f830f33184397c2eb0b651ec502adb63decb28978bdc84b3558d71284c21f)
+DocFiles: COPYING README.md
```

Comment:
```suggestion
DocFiles: COPYING README.md PKG-INFO
PatchScript: <<
    perl -pi -e 's|(/bin/env python)(3)|${1}%type_raw[python]|;' *.py mesonbuild/*.py tools/*.py test*/*/*/*.py
    perl -pi -e "s|(python)(3)(')|\${1}%type_raw[python]\${3}|;" test\ cases/python/[1345]*/meson.build
    perl -pi -e "s|(python)(3)(')|\${1}-%type_raw[python]\${3}|;" test\ cases/python/2*/meson.build
    perl -pi -e "s|(exec =)|add_project_arguments('-I%p/opt/hdf5.v1.10/include', language: ['c', 'cpp'])\n\$1|;" "test cases/frameworks/25 hdf5/meson.build"
    perl -pi -e "s|(exec =)|add_project_link_arguments('-L%p/opt/hdf5.v1.10/lib', language: ['c', 'cpp'])\n\$1|;" "test cases/frameworks/25 hdf5/meson.build"
<<
```
`PKG-INFO` for some concise info, but not mandatory.
I've added those patches to get some more of the tests to pass (see below) and generally try to use the versioned `python` in the scripts where possible.